test/plugin/dropreason/main_linux.go

```diff
@@ -67,31 +67,31 @@ func main() {
 
 	err = tt.Stop()
 	if err != nil {
-		l.Error("Stop failed:%v", zap.Error(err))
+		l.Error("Stop failed:", zap.Error(err))
```

Comment:
nit: these log statement changes should have been in a different PR.

Reply:
Not required, just saw garbled logs so fixed the,